.storybook/preview.js

```diff
@@ -1 +1,2 @@
 import "./stories.css";
+import "../assets/index.css";

```

Comment:
i added this in my PR...and now you have conflicts. sorry about that.but you just need to merge the main branch and is gonna be fine